src/core/annotation.js

```diff
@@ -704,6 +716,51 @@ class Annotation {
     }
   }
 
+  setRotation(mk) {
+    this.rotation = 0;
+    if (mk instanceof Dict) {
+      let angle = mk.get("R") || 0;
+      if (Number.isInteger(angle) && angle !== 0) {
+        // If the angle is negative then angle % 360 is still negative.
+        angle %= 360;
+        if (angle < 0) {
+          angle += 360;
+        }
+        if (angle % 90 === 0) {
+          this.rotation = angle;
+        }
+      }
+    }
+  }
+
+  getRotationMatrix(annotationStorage) {
```

Comment:
Is this necessary for all Annotation-types, or can we perhaps move it into the `WidgetAnnotation`-class instead?